cpp/dolfinx/fem/FiniteElement.cpp

```diff
@@ -66,6 +55,28 @@ FiniteElement::FiniteElement(const ufc_finite_element& ufc_element)
   }
   assert(mesh::cell_dim(_cell_shape) == _tdim);
 
+  static const std::map<ufc_shape, std::string> ufc_to_cell
+      = {{vertex, "point"},
+         {interval, "interval"},
+         {triangle, "triangle"},
+         {tetrahedron, "tetrahedron"},
+         {quadrilateral, "quadrilateral"},
+         {hexahedron, "hexahedron"}};
+  const std::string cell_shape = ufc_to_cell.at(ufc_element.cell_shape);
+
+  const std::string family = ufc_element.family;
+
+  if (family == "mixed element")
```

Comment:
Can you explain this condition?

Reply:
basix does not support mixed elements, so the subelements should be handled separately (I've added this as a comment in the file)